bootstrap.go

```diff
@@ -96,8 +96,26 @@ func startProfiler(options *options) error {
 }
 
 // stop with a graceful shutdown that includes flushing signals.
-func stop() error {
-	tracer.Stop()
-	profiler.Stop()
-	return nil
+func stop(options *options) error {
```

Comment:
I've also seen signature that let the user pass a context/timeout value to the stop method (not that I think one is better than the other.